### DISCUSSION TITLE
statusNotification: warning vs studio 2022 17.2.9

### DISCUSSION THREAD

**Discussion by kxjjuneau** [community/community#38217] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/28900133/199832907-07687552-d848-4002-8657-a64a9b00ef4d.png)  
Im having this issue above, I'm able to successfully validate, but copilot never shows suggestions and every time it tries to the output only spits out the above error message. I've signed out and in again, reinstalled copilot disconected from my vpn, bassically I've tried every solution that Ive seen online but I also havn't seen anyone else with this specific problem

**Comment by iamcjwilliams** [comment]
@kxjjuneau Here are steps for login from a clean state:

1. Close Visual Studio 2022.
2. Delete C:\Users<insert username>\AppData\Local\github-copilot\hosts.json
3. Log out of GitHub in the web browser.
4. Open Visual Studio.

**Reply by kxjjuneau** [reply]
I just tried all that, I closed all instances of vs, deleted the hosts.json, logged out, opened vs 2022 and re-authenticated co-pilot, and still when I try to trigger the inline suggestions it gives the "statusNotification: warning" message

**Comment by Ali-Al1** [comment]
@kxjjuneau Hi, I had the same issue.

Turns out we had to let a couple URLs through the firewall.
https://copilot-telemetry.githubusercontent.com/
https://copilot-proxy.githubusercontent.com/
